airbyte-server/src/main/java/io/airbyte/server/handlers/WorkspacesHandler.java

```diff
@@ -183,6 +184,11 @@ public NotificationRead tryNotification(final Notification notification) {
     return new NotificationRead().status(StatusEnum.FAILED);
   }
 
+  public void setFeedbackDone(final WorkspaceGiveFeedback workspaceGiveFeedback)
+      throws JsonValidationException, ConfigNotFoundException, IOException {
+    configRepository.setFeedback(workspaceGiveFeedback.getWorkspaceId());
```

Comment:
For consistency, this behavior should be in `WorkspacesHandler`

Reply:
Did you make a typo? It is in the WorkspacesHandler already.

Reply:
🤦